Makefile

```diff
@@ -12,7 +12,7 @@ GO=CGO_ENABLED=1 GO111MODULE=on go
 tidy:
 	go mod tidy
 
-unittest:
+unittest: tidy
```

Comment:
I prefer not to have to run tidy every time. It just needs to be run once when the files are out of date.